x-pack/plugins/security_solution/cypress/e2e/timelines/unsaved_timeline.cy.ts

```diff
@@ -145,4 +145,21 @@ describe('Save Timeline Prompts', () => {
     cy.get(ALERTS_PAGE).click();
     cy.get(TIMELINE_SAVE_MODAL).should('not.exist');
   });
+
+  it('Changed and unsaved timeline should NOT prompt when user navigates from the page where timeline is disabled', () => {
+    populateTimeline();
+    waitForTimelineChanges();
+    closeTimelineUsingToggle();
+    openKibanaNavigation();
+    cy.get(MANAGE_PAGE).click();
+    cy.get(APP_LEAVE_CONFIRM_MODAL).should('be.visible');
+    cy.get(MODAL_CONFIRMATION_BTN).click();
+    // now we have come from MANAGE_PAGE where timeline is disabled
+    // to outside app where timeline is not present.
+    // There should be NO confirmation model in that case.
+    openKibanaNavigation();
+    navigateFromKibanaCollapsibleTo(OBSERVABILITY_ALERTS_PAGE);
+    // should not be manage page i.e. successfull navigation
+    cy.url().should('not.contain', MANAGE_URL);
```

Comment:
Maybe we can add a `cy.get(TIMELINE_SAVE_MODAL).should('not.exist');` in this test as well?

Reply:
No harm in added double checks.. Added 👍🏼. Thanks for the feedback.